po/cs.po

```diff
@@ -0,0 +1,560 @@
+msgstr "Kontrolní součet SHA256"
+
+msgid "Compute SHA256 hash sum"
+msgstr "Vypočítat kontrolní součet SHA256"
+
+msgid "SHA384 Checksum"
+msgstr "Kontrolní součet SHA384"
+
+msgid "Compute SHA384 hash sum"
+msgstr "Vypočítat kontrolní součet SHA384"
+
+msgid "SHA512 Checksum"
+msgstr "Kontrolní součet SHA512"
+
+msgid "Compute SHA512 hash sum"
+msgstr "Vypočítat kontrolní součet SHA512"
+
+msgid "MD5 Checksum"
+msgstr "Kontrolní součet MD5"
+
+msgid "Compute MD5 hash sum"
+msgstr "Vypočítat kontrolní součet MD5"
+
+msgid "Base64 Encode"
+msgstr "Kódování Base64"
+
+msgid "Encode text to Base64"
+msgstr "Kódování textu do Base64"
+
+msgid "Base64 Decode"
+msgstr "Dekódování Base64"
+
+msgid "Decode text from Base64"
+msgstr "Dekódování textu z Base64"
+
+msgid "Remove Trailing"
+msgstr "Odstranit koncování"
+
+msgid "Remove trailing whitespaces and newlines"
+msgstr "Odstranit Koncové mezery a nové řádky"
+
+msgid "Trim Lines"
+msgstr "Oříznout řádky"
+
+msgid "Remove leading and trailing whitespaces and newlines"
+msgstr "Odstranit načítání a koncové mezery a nové řádky"
+
+msgid "Count Symbols"
+msgstr "Počet symbolů"
+
+msgid "Count how many symbols are in text"
+msgstr "Spočítat, kolik symbolů je v textu"
+
+msgid "Count Lines"
+msgstr "Počet řádků"
+
+msgid "Count how many lines are in text"
+msgstr "Spočítat, kolik řádků je v textu"
+
+msgid "Count Words"
+msgstr "Počet slov"
+
+msgid "Count how many words are in text"
+msgstr "Spočítat, kolik slov je v textu"
+
+msgid "Prettify JSON"
+msgstr "Zkrášlit JSON"
+
+msgid "Parse JSON and prettify it"
+msgstr "Zpracovat JSON a zkrášlit jej"
+
+msgid "Minify JSON"
+msgstr "Minifikovat JSON"
+
+msgid "Parse JSON and minify it"
+msgstr "Zpracovat JSON a minifikovat jej"
+
+msgid "Escape String"
+msgstr "Únikový řetězec"
+
+msgid "Replace special characters with escape sequences"
+msgstr "Nahradit speciální znaky s uniklými sekvencemi"
+
+msgid "Escape HTML"
+msgstr "Únikový HTML"
+
+msgid "Replace special characters with HTML-safe sequences"
+msgstr "Nahradit speciální znaky pomocí zabezpečených HTML sekvencí"
+
+msgid "Escape URL"
+msgstr "Úniková URL adresa"
+
+msgid "Replace special characters with URL-safe sequences"
+msgstr "Nahradit speciální znaky pomocí zabezpečených URL sekvencí"
+
+msgid "Unescape String"
+msgstr "Unescape String"
+
+msgid "Replace escape sequences with characters"
+msgstr "Nahradit únikové sekvence znaky"
+
+msgid "Unescape HTML"
+msgstr "Unescape HTML"
+
+msgid "Replace HTML sequences with characters"
+msgstr "Nahradit HTML sekvence znaky"
+
+msgid "Unescape URL"
+msgstr "Unescape URL"
+
+msgid "Replace URL sequences with characters"
+msgstr "Nahradit URL sekvence znaky"
+
+msgid "Convert JSON to YAML"
+msgstr "Převést JSON na YAML"
+
+msgid "Parse JSON and convert it to YAML document"
+msgstr "Zpracovat JSON a převést jej na YAML dokument"
+
+msgid "Convert YAML to JSON"
+msgstr "Převést YAML na JSON"
+
+msgid "Parse YAML document and conver it to JSON"
```

Comment:
This typo has been fixed, it needs to be `"Parse YAML document and convert it to JSON"` now